### ISSUE TITLE
Video not being transmitted

### ISSUE THREAD

**Issue by stephenhibbert** [awslabs/aws-iot-greengrass-edge-connector-for-kinesis-video-stream#25]
Hello, could you help me to diagnose an issue I have?

- I have checked the IoT role permissions, that's all correct
- I have confirmed that the RTSP stream UR is correct and reachable from ffmpeg on the network
- I see no data for the metic for PutMedia but there is some metrics for HLR and MPEG-DASH requests
- See component logs attached

<img width="1016" alt="Pasted Graphic 2" src="https://user-images.githubusercontent.com/11485349/168279593-a638b924-6afa-4f0c-a7f8-9e4fc4518118.png">
<img width="1141" alt="Pasted Graphic 1" src="https://user-images.githubusercontent.com/11485349/168279608-d77cf666-973b-475b-a70f-55573d6f9b58.png">
[logs.txt](https://github.com/awslabs/aws-iot-greengrass-edge-connector-for-kinesis-video-stream/files/8686847/logs.txt)
 
Any pointers would be welcome!

**Comment by stephenhibbert** [comment]
Solved my own issue - it was a dependency issue with the gstreamer libs. Is there any we this could be checked and output to the component logs to make debugging dependency issues easier?

**Comment by lightbulbkhuang** [comment]
Hi @stephenhibbert,

May I know what dependency issue of the GStreamer libs you met?
Thank you.

**Comment by stephenhibbert** [comment]
Of course, I simply hadn't installed the libs as mentioned in the docs pointing to this: https://gstreamer.freedesktop.org/documentation/installing/on-linux.html?gi-language=c - however I didn't see any clues in the logs to suggest this was the issue. Installing gstreamer properly fixed the issue.
